### PR TITLE
Disco Pane content update - Jan. 2017

### LINKED ISSUE #4198: Disco Pane content update - Jan. 2017
For the **19 January** push, can we please REMOVE:

- Awesome Screenshot Plus
- All three themes (Ice Flower, Les Éléphants, Owl First Snow)

... and REPLACE them with Emoji Keyboard and three new themes, respectively (note full copy included):

**Emoji expression** 
with [Emoji Keyboard by Harry N.](https://addons.mozilla.org/firefox/addon/emojikeyboard/)
Dozens of amazing emojis for every occasion—always just one click away. 

[themes]

Symphony of Colors by [rositza212](https://addons.mozilla.org/firefox/addon/symphony-of-colors/)

Stained Glass Fractal by [Guitarzan](https://addons.mozilla.org/firefox/addon/stained-glass-fractal/)

Blue Twirl by [RLC ♥†](https://addons.mozilla.org/firefox/addon/blue-twirl/)

### PR DESCRIPTION
**Pull request by diox**
Fix #4198

### CODE REVIEW

File: src/olympia/discovery/data.py [hunk 1]
```diff
@@ -12,8 +12,8 @@ def __init__(self, *args, **kwargs):
 # At the moment the disco pane items are hardcoded in this file in the repos,
 # which allows us to integrate in our translation workflow easily.
 discopane_items = [
-    # Theme: Ice Flower
-    DiscoItem(addon_id=265447),
+    # Theme: Symphony of colors
```
Comment (EnTeQuAk) [inline src/olympia/discovery/data.py]: The github-issue writes "Colors" instead of "colors" - for now I always went with the copy that we get from Scott.

Comment (diox) [inline src/olympia/discovery/data.py]: mmm, without translating it, just forcing the `addon_name` ?

Comment (EnTeQuAk) [inline src/olympia/discovery/data.py]: Correct

Comment (diox) [inline src/olympia/discovery/data.py]: https://github.com/mozilla/addons-server/pull/4204/commits/21a292c44a5ea86e4cac8e6fbd899e3f0cc83b69

Comment (EnTeQuAk) [inline src/olympia/discovery/data.py]: NIce, thanks! lgtm

**Pull request merged**